### PR TITLE
More renames of RaftID to RangeID and Range to Replica

### PR DESCRIPTION
**Pull request by bdarnell**
More mostly-automated work towards #1864.

### CODE REVIEW

**Comment by tschottdorf** [comment]
Under the assumption that `/raft\s?id/i` and `/Range/` in `find . '*.go'` come back as expected, LGTM. Or is there something to actually review?

File: storage/client_raft_test.go [hunk 1]
```diff
@@ -90,30 +90,30 @@ func TestStoreRecoverFromEngine(t *testing.T) {
...
-		splitArgs := adminSplitArgs(proto.KeyMin, splitKey, raftID, store.StoreID())
+		splitArgs := adminSplitArgs(proto.KeyMin, splitKey, rangeID, store.StoreID())
 		if _, err := store.ExecuteCmd(context.Background(), &splitArgs); err != nil {
 			t.Fatal(err)
 		}
-		raftID2 = store.LookupRange(key2, nil).Desc().RangeID
-		if raftID2 == raftID {
+		rangeID2 = store.LookupReplica(key2, nil).Desc().RangeID
+		if rangeID2 == rangeID {
 			t.Errorf("got same raft id after split")
```
Comment (tschottdorf) [inline storage/client_raft_test.go]: s/raft/range/

**Comment by bdarnell** [comment]
Yeah, nothing to really review. `raft.?id` is now gone from the go codebase (except for `raftIdleTimeout`). I'm proceeding more slowly on the `Range` to `Replica` renaming because in many cases we do want to keep the `Range` name (e.g. `RangeDescriptor` refers to the range, not the replica).

**Comment by bdarnell** [comment]
@BramGruneir Take a look at the last commit; we were using `raft_id` in the frontend but that field has been renamed.

**Comment by mrtracy** [comment]
LGTM, I verified the front-end changes work.

**Pull request merged**